doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -536,17 +536,17 @@ Figure 3-3 below shows the main building blocks of a Kubernetes networking solut
    service meshes are outside the scope of the infrastructure layer of this
    architecture.
 
-.. image:: ../figures/ch03_networking.png
-   :alt: "Figure 3-3: Kubernetes Networking Architecture"
+.. figure:: ../figures/ch03_networking.png
+   :alt: Kubernetes Networking Architecture
+   :name: Kubernetes Networking Architecture
 
+   Kubernetes Networking Architecture
 
 .. raw:: html
```

Comment:
line 547 and 548 are comments -- can we just use rst comments and not raw html

Reply:
Done in [fea5914](https://github.com/cntt-n/CNTT/pull/3065/commits/fea59141befa0e4eb69b928e1807f3f6acab1556)